### PR TITLE
Add file that summarizes the release process.

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+1. Create release branch `git checkout -b release/0.10.0`
+1. Update the version in `package.json`
+1. Update `CHANGELOG.md`, move changes under the new version, don't forget to update at bottom of file.
+1. Commit the [changes](https://github.com/comit-network/comit-js-sdk/pull/103/files) with message `Release 0.10.0`
+1. Push the changes
+1. Create PR, wait for checks and approvals
+1. (don't merge the PR, add the tags first; if merged you might have to remove the branch yourself later)
+1. Add tags `git tag 0.10.0`
+1. Push the tags ` git push --tags`
+1. Got to the `Checks` on Github and wait till the release action to npmjs.com was done.
```
Comment (da-kami) [inline RELEASE.md]:
```suggestion
1. Go to the `Checks` section in the PR on Github and wait till the release action to npmjs.com was done.
```

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Releasing the SDK
+
+This document summarizes the steps to release the `comit-js-sdk`.
+The steps below were recorded when releasing version `0.10.0`.
+You of course have to replace `0.10.0` with your version to be released!
```
Comment (luckysori) [inline RELEASE.md]: 🔧 I would just remove this line and use a stub like `X.Y.Z` or `<version-number>`.

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Releasing the SDK
```
Comment (luckysori) [inline RELEASE.md]: ❓ Should we mention that we follow SemVer and [link](https://semver.org/) to it?

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+This document summarizes the steps to release the `comit-js-sdk`.
+The steps below were recorded when releasing version `0.10.0`.
+You of course have to replace `0.10.0` with your version to be released!
+
+1. Open changelog and decide new version number based on changes.
```
Comment (luckysori) [inline RELEASE.md]: ❓ Should we mention that we follow SemVer and [link](https://semver.org/) to it?

File: RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Releasing the SDK
+
+This document summarizes the steps to release the `comit-js-sdk`.
+The steps below were recorded when releasing version `0.10.0`.
+You of course have to replace `0.10.0` with your version to be released!
+
+1. Open changelog and decide new version number based on changes. 
+1. Create release branch `git checkout -b release/0.10.0`
+1. Update the version in `package.json`
+1. Update `CHANGELOG.md`, move changes under the new version, don't forget to update at bottom of file.
```
Comment (luckysori) [inline RELEASE.md]: 🔧 
```suggestion
1. Update `CHANGELOG.md`, move unreleased changes under the new version and update links at the bottom of the file.
```

File: RELEASE.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Releasing the SDK
+
+This document summarizes the steps to release the `comit-js-sdk`.
+The steps below were recorded when releasing version `0.10.0`.
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
```

File: RELEASE.md (hunk 3 above)
Comment (D4nte) [inline RELEASE.md]:
```suggestion
# Releasing comit-sdk
```

File: RELEASE.md (hunk 2 above)
Comment (D4nte) [inline RELEASE.md]:
```suggestion
Replace `X.Y.Z` with the new version.
```

File: RELEASE.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+You of course have to replace `0.10.0` with your version to be released!
+
+1. Open changelog and decide new version number based on changes.
+1. Create release branch `git checkout -b release/0.10.0`
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
1. Create release branch `git checkout -b release/X.Y.Z`
```

File: RELEASE.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+1. Update the version in `package.json`
+1. Update `CHANGELOG.md`, move changes under the new version, don't forget to update at bottom of file.
+1. Commit the [changes](https://github.com/comit-network/comit-js-sdk/pull/103/files) with message `Release 0.10.0`
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
1. Commit the [changes](https://github.com/comit-network/comit-js-sdk/pull/103/files) with message `Release X.Y.Z`
```

File: RELEASE.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+1. Push the changes
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
```

File: RELEASE.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+1. Create PR, wait for checks and approvals
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
1. Create a Pull Request, wait for checks and approvals,
```

File: RELEASE.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+1. (don't merge the PR, add the tags first; if merged you might have to remove the branch yourself later)
+1. Add tags `git tag 0.10.0`
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
1. Tag the release commit `git tag X.Y.Z`
```

File: RELEASE.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+1. Push the tags ` git push --tags`
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
1. Push the tag `git push --tags`,
```

File: RELEASE.md [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+1. Go to the `Checks` section in the PR on Github and wait till the release action to npmjs.com was done.
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
1. Go to the `Checks` section in the PR on Github and wait until the release action to npmjs.com was done,
```

File: RELEASE.md [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+1. Go to npmjs.com, search for `comit-sdk` and check if it was released correctly there.
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
1. Go to https://npmjs.com, search for `comit-sdk` and check if it was released correctly there,
```

File: RELEASE.md [hunk 15]
```diff
@@ -0,0 +1,19 @@
...
+1. (Remove the remote branch if necessary)
```
Comment (D4nte) [inline RELEASE.md]:
```suggestion
1. Remove the remote branch if you merged before pushing.
```

File: RELEASE.md [hunk 16]
```diff
@@ -0,0 +1,17 @@
+# Releasing comit-sdk
+
+This document summarizes the steps to release the `comit-js-sdk`.
+Replace `X.Y.Z` with the new version.
+
+1. Open changelog and decide new version number based on changes.
```
Comment (da-kami) [inline RELEASE.md]:
```suggestion
1. Open changelog and [decide new version number](https://semver.org/) based on changes.
```

**Comment by D4nte** [comment]
@da-kami   Please squash to one commit once you are happy.

**Pull request merged**